spyder/utils/fixtures.py

```diff
@@ -41,3 +43,14 @@ def fin():
 
     request.addfinalizer(fin)
     return CONF
+
+@pytest.fixture
+def editorbot(qtbot):
+    widget = codeeditor.CodeEditor(None)
+    widget.setup_editor(linenumbers=True, markers=True, tab_mode=False,
+                         font=QFont("Courier New", 10),
+                         show_blanks=True, color_scheme='Zenburn')
+    widget.setup_editor(language='Python')
```

Comment:
Both are right, I forgot to change that bot for the one that I'm using and that is in `test_editor.py` 😖